### ISSUE TITLE
Change app theme dynamically

### ISSUE THREAD

**Issue by sriramr98** [fossasia/open-event-droidgen#2398]
**Change app theme dynamically**

We should add a color chooser in Settings which can enable the user to choose the theme of the app dynamically. Most people using the app are going to be tech enthusiasts and they'd prefer to have the option to choose themes instead of the default red.

**Steps to reproduce it**

I will be using this library [Colorful](https://github.com/garretyoder/Colorful) to implement dynamic theming in the application. The library itself is simple to use.

**Would you like to work on the issue?**

Yes

**Comment by iamareebjamal** [comment]
There's an option to pick the accent color and theme in the app generator. So it's not only red, it's what the app generator chooses. The effect is similar to what other event apps do. Like Google I/O or F8, they have a particular theme

**Comment by sriramr98** [comment]
Yeah but that's the theme that the company chooses and it's static. The user has no say in the theme. What if the user doesn't like red or blue and wants green ?  What is the user has selective cor blindness? In those cases I feel it's important to add a setting so which will contribute towards personalisation. And thats the main attraction of android. Personalisation.

**Comment by iamareebjamal** [comment]
The complexity of designing layouts when presented with lots of choices to the user increases manifolds. This can be taken up when the app at least supports dark mode

**Comment by sriramr98** [comment]
Yeah, we can take this up after adding dark theme. Dark mode can also be easily added using this library. We will get to this after kotlin refactor is done
